ConditionalTranslation.md

```diff
@@ -114,13 +113,18 @@ Refer to the [updated grammar appendix](#appendix-updated-grammar) for the list
    * function call statements
    * const assertion statements
 
+> [!WARNING]
+> We added attributes on assignment and increment/decrement statements. This change makes the WESL grammar no longer [LR(1)](https://en.wikipedia.org/wiki/LR_parser). See [`wesl-rs#162`](https://github.com/wgsl-tooling-wg/wesl-rs/issues/162) for details.
```

Comment:
I know too little about the different kinds of parsers. Is wesl still LALR(1), or does it also break that?